pkg/apis/flowcontrol/validation/validation.go

```diff
@@ -20,15 +20,51 @@ import (
 	"fmt"
 	"strings"
 
+	apiequality "k8s.io/apimachinery/pkg/api/equality"
 	"k8s.io/apimachinery/pkg/api/validation"
 	apimachineryvalidation "k8s.io/apimachinery/pkg/api/validation"
+	"k8s.io/apimachinery/pkg/conversion"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/validation/field"
+	"k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap"
 	"k8s.io/apiserver/pkg/util/shufflesharding"
 	apivalidation "k8s.io/kubernetes/pkg/apis/core/validation"
 	"k8s.io/kubernetes/pkg/apis/flowcontrol"
+	"k8s.io/kubernetes/pkg/apis/flowcontrol/install"
 )
 
+var mandatoryFlowSchemas = make(map[string]*flowcontrol.FlowSchemaSpec)
+var mandatoryPriorityLevels = make(map[string]*flowcontrol.PriorityLevelConfigurationSpec)
+
+func init() {
```

Comment:
What about making a parallel "internal" bootstrapping package and putting this conversion code there? E.g., "k8s.io/kubernetes/pkg/apis/flowcontrol/bootstrap"?

Reply:
done